src/olympia/blocklist/tests/test_admin.py

```diff
@@ -1361,7 +1361,7 @@ def test_signoff_approve(self):
         log_entry = LogEntry.objects.last()
         assert log_entry.user == user
         assert log_entry.object_id == str(mbs.id)
-        other_obj = addon_factory(id=mbs.id)
+        other_obj = addon_factory()
```

Comment:
Seems like a random change in an unrelated part of the codebase? iirc it's using the same pk as mbs (the `BlockSubmission` instance - previously `MultiBlockSubmission`) to test the LogEntry shown on the page are being correctly filtered on content type, not just id.